docs/changelogs/vNEXT.rst

```diff
@@ -24,6 +24,14 @@ Link to migration guide
 New features
 ==============
 
+Windows Support
+===============
+The three oso libraries (python, ruby and java) all now work on Windows.
+
+Musl Python Builds
+===========
+Python linux musl-based wheels (for alpine and other musl based distros) are built and downloadable from the release in Github.
```

Comment:
```suggestion
musl builds for Python
======================

musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from [the releases page on GitHub](https://github.com/osohq/oso/releases).
```

Reply:
ah crap, that's a markdown-style link

Reply:
We're going to publish it as `oso-alpine` right?

Reply:
Or `oso-musl` I forget.

Reply:
updated (I think)

Reply:
It's not published to pypi at all yet. It's just available as a download on the Github release.